backend/api/views.py

```
@@ -1,96 +1,75 @@
-    def get_queryset(self):
-        return SurveyResponse.objects.filter(user=self.request.user)
+    def post(self, request):
+        viewer_profile, created = ViewerProfile.objects.get_or_create(user=request.user)
+        
+        serializer = OnboardingSerializer(viewer_profile, data=request.data, partial=True)
+        
+        if serializer.is_valid():
+            serializer.validated_data['onboarding_completed'] = True
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_200_OK)
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)